cli_tools/gce_image_publish/publish/publish.go

```diff
@@ -308,6 +308,10 @@ func publishImage(p *Publish, img *Image, pubImgs []*compute.Image, skipDuplicat
 			}
 		}
 
+		if pubImg.Family == "" {
```

Comment:
This should probably be up at the check on line 295 so we don't delete images either. And check for img.Family == "" to make the logic clearer

Reply:
My argument would be then the `DeleteAfter` field would only work within the same image family. I still want to control lifecycle of all the images in the project though.

Reply:
Hmm, I guess I can see that, its a bit odd running this with no family as is so I think this shouldn't be an issue

Reply:
thanks!

Most of the GKE OS images doesn't have associated family, and I'm exporting & publishing them one by one :-)